lib/mpl_toolkits/axisartist/angle_helper.py

```diff
@@ -284,7 +284,7 @@ def __call__(self, direction, factor, values):
                 return r
 
         else:  # factor > 3600.
-            return [r"$%s^{\circ}$" % (str(v),) for v in ss*values]
+            return [r"$%s^{\circ}$" % v for v in ss*values]
```

Comment:
```suggestion
            return [r"$%s^{\circ}$" % (v, ) for v in ss*values]
```

Mild paranoia about `%` formatting being too helpful.

Reply:
`ss*values` is clearly a numpy array, so no chance of `v` being a tuple.